NEWS.md

```diff
@@ -1,6 +1,8 @@
 # Version 1.0.9000
-* Renamed `fromPrior` to `from_prior` argument of `mcmc` method.
-* Removed `program` and `verbose` arguments from `mcmc` method.
+* Added logger feature. It consists of four user-end functions:
+  `verbose_logging`, `disable_logging`, `is_logging_verbose`, `log_trace`.
```

Comment:
```suggestion
  `enable_logging`, `disable_logging`, `is_logging_verbose`, `log_trace`.
```
to be more analog between "disable" and "enable" ?

Reply:
Tx @danielinteractive, maybe then also change `is_logging_verbose` to `is_logging_enabled`?